.editorconfig

```diff
@@ -13,7 +13,7 @@ insert_final_newline = true
 trim_trailing_whitespace = true
 indent_style = tab
 
-[{.jshintrc,*.json,*.yml}]
+[{.jshintrc,*.yml}]
```

Comment:
I think we can remove `.jshintrc` as that file is already using tabs.

Reply:
did in https://github.com/Automattic/jetpack/pull/11507/commits/f54fd2bf873ac89ea72961297157365285f78b9f